.travis.yml

```diff
@@ -13,8 +13,9 @@ script:
   - make test
   - bundle exec codeclimate-test-reporter
 env:
-  - RACK_ENV=test
-  - RAILS_ENV=test
+- RACK_ENV=test
+- RAILS_ENV=test
+- COVERAGE=true
```

Comment:
Check out the two builds for this PR: https://travis-ci.org/18F/identity-dashboard/builds/208771094

and then after my PR https://travis-ci.org/18F/identity-dashboard/builds/208784528